src/Components/Pronunciation/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Button } from "antd";
+import { SoundOutlined } from "@ant-design/icons";
+import "./Pronunciation.scss";
+
+const Pronunciation = ({ sectionData }) => {
+  const getAudioPath = (sectionData) => {
+    if (sectionData.audio.length > 0) {
+      return sectionData.audio.find((audio) => audio !== "");
+    }
+  };
+  const playAudio = (e) => {
+    e.stopPropagation();
+    var audio = new Audio();
+    audio.src = getAudioPath(sectionData);
+    audio.play();
+  };
+
+  return (
+    sectionData.audio.length > 0 && (
```

Comment:
Your logic to show the pronunciation button is buggy. What if the `audio` array is `[""]`?